packages/flutter_tools/lib/src/reporting/crash_reporting.dart

```diff
@@ -26,15 +26,61 @@ const String _kStackTraceFileField = 'DartError';
 /// it must be supplied in the request.
 const String _kStackTraceFilename = 'stacktrace_file';
 
+class CrashDetails {
+  CrashDetails({
+    @required this.command,
+    @required this.error,
+    @required this.stackTrace,
+    @required this.doctorText,
+  });
+
+  String command;
+  dynamic error;
+  StackTrace stackTrace;
+  String doctorText;
```

Comment:
Can you make these final and private?

Reply:
I can make them final.  Private doesn't make sense; the point is for these fields to be extracted by whatever the `CrashDetails` object is passed to.